dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/CompanyManager.kt

```diff
@@ -45,39 +46,75 @@ class CompanyManager : CompanyManagerInterface {
         return companyDataPerCompanyId[companyId]!!
     }
 
-    override fun searchCompanies(searchString: String, onlyCompanyNames: Boolean): List<StoredCompany> {
-        val allCompanies = companyDataPerCompanyId.values.toList()
+    override fun searchCompanies(
+        searchString: String,
+        onlyCompanyNames: Boolean,
+        dataTypeFilter: Set<DataType>,
+        stockIndexFilter: Set<StockIndex>
+    ): List<StoredCompany> {
+        var companies = companyDataPerCompanyId.values.toList()
 
-        return if (searchString == "") {
-            allCompanies
-        } else if (onlyCompanyNames) {
-            filterCompaniesByName(searchString, allCompanies)
+        if (onlyCompanyNames) {
```

Comment:
könnte man auch so schreiben: 
companies = if (onlyCompanyNames) {
            filterCompaniesByNameOnly(searchString, companies)
        } else {
            filterCompaniesByNameAndIdentifier(searchString, companies)
        }

Reply:
Danke für den Hinweis. Nehme ich so in meinen nächsten Commit auf